btrack/utils.py

```diff
@@ -375,7 +383,10 @@ def tracks_to_napari(tracks: list, ndim: int = 3, replace_nan: bool = True):
     """
     # TODO: arl guess the dimensionality from the data
     assert ndim in (2, 3)
-    t_header = ["ID", "t"] + ["z", "y", "x"][-ndim:]
+    coords_axes = ["z", "y", "x"] if coords_axes is None else coords_axes
```

Comment:
Then you could get rid of this line and just make sure that:
```python
if not all([axis in ("x", "y", "z") for axis in coords_axes]):
  raise ValueError
```
just to check that these are only spatial dims?